src/main/java/edu/cornell/kfs/tax/businessobject/DvDisbursementView.java

```diff
@@ -0,0 +1,47 @@
+package edu.cornell.kfs.tax.businessobject;
+
+import org.kuali.kfs.krad.bo.TransientBusinessObjectBase;
+import org.kuali.rice.core.api.util.type.KualiInteger;
+
+/**
+ * Convenience lightweight BO to allow for ORM tools to map to the DvDisbursementView
+ */
+public class DvDisbursementView extends TransientBusinessObjectBase {
```

Comment:
The SQL for the related view also has a column for the Payment Group ID. Should that be added to this BO as well?

If a property for that gets added here, make sure it gets included in the OJB XML file.